wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitIndexChangedDetector.java

```diff
@@ -84,7 +91,9 @@ public void stopWatcher() {
   private PathMatcher matcher() {
```

Comment:
Add comment about the purpose of the matcher.

Reply:
done